workflow/rocoto/rocoto.py

```diff
@@ -146,7 +146,8 @@ def _create_innermost_task(task_dict: Dict[str, Any]) -> List[str]:
     strings.append(f'\t<walltime>{walltime}</walltime>\n')
     strings.append(f'\t<nodes>{nodes}:ppn={ppn}:tpp={threads}</nodes>\n')
     if memory is not None:
-        strings.append(f'\t<memory>{memory}</memory>\n')
+        if memory != 'None':
+            strings.append(f'\t<memory>{memory}</memory>\n')
```

Comment:
Revert

Reply:
Since "unset memory" worked in ".resource.AWSPW", reverted this change.